WordPress/src/main/java/org/wordpress/android/ui/SlidingRelativeLayout.java

```diff
@@ -0,0 +1,48 @@
+package org.wordpress.android.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewTreeObserver;
+import android.widget.RelativeLayout;
+
+public class SlidingRelativeLayout extends RelativeLayout {
+
+    private float xFraction = 0;
+
+    public SlidingRelativeLayout(Context context) {
+        super(context);
+    }
+
+    public SlidingRelativeLayout(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    private ViewTreeObserver.OnPreDrawListener preDrawListener = null;
+
+    // This implementation fixes the first frame not being translated: http://trickyandroid.com/fragments-translate-animation/
+    public void setXFraction(float fraction) {
+        this.xFraction = fraction;
```

Comment:
Nitpicking here: Use of `this` is redundant, especially when naming conventions are followed, in which case the name `mXFraction` would already make it clear that it's the instance variable we want to assign to.
It boils down to a matter of personal coding style (and we don't have a very consistent pattern about it in the rest of the app) so I'll leave this totally to you. Cheers!